### DISCUSSION TITLE
Possible to prevent the auto dimming on OLED monitors?

### DISCUSSION THREAD

**Discussion by kast231** [JPersson77/LGTVCompanion#254] [Ideas]
Hello, great app.  thank you all for developing it.  I was hoping you maybe able to add a feature for OLED owners.  

The correct behavior on HDMI- PC connection is the TV will dim to prevent burn in.  Mouse strokes are not big enough to sensed by the TV, you need to do a larger landscape change on the monitor for the TV to sense pixels changing.  I am not sure how many LEDs need to be used to trigger the tv out of autodim mode.  

I wonder if there is a way to work a feature that cancels autodim by sending a larger signal or some sorts, or a way to disable it all together or extend the idle time into the LGTV Companion app.  It would be very helpful.

**Comment by dechamps** [comment]
> a way to disable it all together or extend the idle time into the LGTV Companion app

I'm not sure if the LG protocol makes it possible to disable TPC directly, but you can already use LGTV Companion to [open the service menu](https://github.com/JPersson77/LGTVCompanion/blob/70483957240e4f54c4ce9b38bb5168c0abceaeb4/Docs/Commandline.md#accessing-the-service-menu-lgtv-companion-and-lgtvcli) and disable TPC there.

**Comment by JPersson77** [comment]
I added some switches the latest version (v4.4.0) for controlling GSR/TPC directly. GSR and TPC is the automatic brightness limiter (ABL). It should work for most models, but forC4 and later I only heard it anecdotally. Let me know how it goes

-servicemenu_tpc_enable
-servicemenu_tpc_disable
-servicemenu_gsr_enable
-servicemenu_gsr_enable
